src/visualization/types/Map/view.tsx

```diff
@@ -21,51 +27,97 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
     detectCoordinateFields,
     mapStyle,
   } = properties
-  const tileServerConfiguration = {
-    tileServerUrl: '',
-    bingKey: '',
-  }
 
   const {lat, lon} = properties.center
 
-  let calculatedGeoCoordinates = {
+  const [errorCode, setErrorCode] = useState<string>('')
+  const [mapToken, setMapToken] = useState<string>('')
+  const [geoCoordinates, setGeoCoordinates] = useState<Coordinates>({
     lat,
     lon,
-  }
-  try {
-    calculatedGeoCoordinates = getGeoCoordinates(result.table, 0)
-  } catch (err) {
-    const error =
-      'Map type is not supported with the data provided: Missing latitude/longitude values'
+  })
+
+  useEffect(() => {
+    const getToken = async () => {
+      try {
+        const {token} = await getMapToken()
+        setMapToken(token)
+      } catch (err) {
+        setErrorCode('MapServiceUnavailable')
+      }
+    }
+    const getCoordinates = () => {
+      let calculatedGeoCoordinates = {
+        lat,
+        lon,
+      }
+      try {
+        calculatedGeoCoordinates = getGeoCoordinates(result.table, 0)
+        setGeoCoordinates(calculatedGeoCoordinates)
+      } catch (err) {
+        setErrorCode('InvalidCoordinates')
+      }
+    }
+    getCoordinates()
+    getToken()
+  }, [])
 
-    return (
-      <div className="panel-resizer--error" data-testid="geoplot-error">
-        {error}
-      </div>
-    )
+  let error = ''
+
+  const getMapboxUrl = () => {
+    if (mapToken) {
+      return `https://api.mapbox.com/styles/v1/influxdata/cklspghhd1opp17mmz0yk7rcy/tiles/256/{z}/{x}/{y}@2x?access_token=pk.eyJ1IjoiaW5mbHV4ZGF0YSIsImEiOiJja2hndnM0eWQwNTVtMzRwZnd5cmQxNHhoIn0.IEAjIguiM67ql4mhiD88rw`
+    }
+    return ''
   }
 
-  return (
-    <Plot
-      config={{
-        table: result.table,
-        showAxes: false,
-        layers: [
-          {
-            type: 'geo',
-            lat: calculatedGeoCoordinates.lat,
-            lon: calculatedGeoCoordinates.lon,
-            zoom,
-            allowPanAndZoom,
-            detectCoordinateFields,
-            mapStyle,
-            layers,
-            tileServerConfiguration: tileServerConfiguration,
-          },
-        ],
-      }}
-    />
-  )
+  switch (errorCode) {
+    case 'MapServiceUnavailable':
+      error =
+        'Map type is having issues connecting to the server. Please try again later.'
```

Comment:
can you write up a ticket to look over the error messages with product / design / docs? I'm not sure exactly who we need to talk to, maybe ask Kristina who should be included

Reply:
Will create an additional issue to track this.